### PR TITLE
Added application worker example

### PR DESCRIPTION
**Pull request by Mazyod**
Hello!

Given someone in your skill and experience, I can see why you missed adding such an example. It took someone new to Elixir quite a while to figure out. I knew there had to be a way, so I decided to find it and propose a PR instead of opening an issue :+1: 

Hope this will help someone in my situation!

### CODE REVIEW

**Comment by ismaelga** [comment]
Shouldn't it be `Calculator` instead of `ErlportTest.GameLogic`?

**Comment by Mazyod** [comment]
@ismaelga I just realized and updated that!

File: README.md [hunk 1]
```diff
@@ -166,6 +166,21 @@ end
 Calculator.start_link(x, name: :foo)
 ```
 
+This will allow you to pass in the actor process name when instantiated as a worker by the application like so:
```
Comment (sasa1977) [inline README.md]: Thank you for the PR!

I would propose to condense this, and add only supervisor application to the block above. So immediately below line 166, you could add something like:

```elixir
# Or in the supervisor specification:
Supervisor.start_link(
  [
    worker(Calculator, [x, [name: :foo]]),
    # ...
  ]
)
```

This seems consistent with the previous code snippet, and I believe it's enough information. What do you think?

Comment (Mazyod) [inline README.md]: Ah, makes sense. For me, any example that has the `worker` keyword works :) I'll make this change and squash the commits in a bit

**Comment by sasa1977** [comment]
Thanks! I did a small change directly in the merge commit. Hope you don't mind :-)

**Comment by Mazyod** [comment]
@sasa1977 I appreciate the quality standard you set!

**Pull request merged**